src/orders/components/OrderDraftDetailsDatagrid/datagrid.ts

```diff
@@ -21,62 +21,52 @@ import { useOrderLineDiscountContext } from "@dashboard/products/components/Orde
 import getOrderErrorMessage from "@dashboard/utils/errors/order";
 import { GridCell, Item } from "@glideapps/glide-data-grid";
 import { DefaultTheme, useTheme } from "@saleor/macaw-ui/next";
-import { useMemo } from "react";
 import { IntlShape, useIntl } from "react-intl";
 
 import { lineAlertMessages } from "../OrderDraftDetailsProducts/messages";
 import { columnsMessages } from "./messages";
 
-export const useColumns = () => {
-  const emptyColumn = useEmptyColumn();
-  const intl = useIntl();
-
-  const availableColumns = useMemo(
-    () => [
-      emptyColumn,
-      {
-        id: "product",
-        title: intl.formatMessage(columnsMessages.product),
-        width: 300,
-      },
-      {
-        id: "sku",
-        title: "SKU",
-        width: 150,
-      },
-      {
-        id: "variantName",
-        title: intl.formatMessage(columnsMessages.variantName),
-        width: 150,
-      },
-      {
-        id: "quantity",
-        title: intl.formatMessage(columnsMessages.quantity),
-        width: 80,
-      },
-      {
-        id: "price",
-        title: intl.formatMessage(columnsMessages.price),
-        width: 150,
-      },
-      {
-        id: "total",
-        title: intl.formatMessage(columnsMessages.total),
-        width: 150,
-      },
-      {
-        id: "status",
-        title: "Status",
-        width: 250,
-      },
-    ],
-    [emptyColumn, intl],
-  );
-
-  return {
-    availableColumns,
-  };
-};
+export const orderDraftDetailsStaticColumnsAdapter = (
+  emptyColumn: AvailableColumn,
+  intl: IntlShape,
+) => [
+  emptyColumn,
+  {
+    id: "product",
+    title: intl.formatMessage(columnsMessages.product),
+    width: 300,
+  },
+  {
+    id: "sku",
+    title: "SKU",
+    width: 150,
+  },
+  {
+    id: "variantName",
+    title: intl.formatMessage(columnsMessages.variantName),
+    width: 150,
+  },
+  {
+    id: "quantity",
+    title: intl.formatMessage(columnsMessages.quantity),
+    width: 80,
+  },
+  {
+    id: "price",
+    title: intl.formatMessage(columnsMessages.price),
+    width: 150,
+  },
+  {
+    id: "total",
+    title: intl.formatMessage(columnsMessages.total),
+    width: 150,
+  },
+  {
+    id: "status",
+    title: "Status",
```

Comment:
**issue**: untranslated message. I see it was like that before, but I think this should be fixed